### PR TITLE
Delete autosuggest examples and redirect to autocomplete

### PR DESCRIPTION
**Pull request by caroqliu**
- Removes experiment tag from `amp-autocomplete` and ComboBox examples as this deletion is contingent upon amp-autocomplete being launched.

### CODE REVIEW

File: platform/config/amp-dev-redirects.yaml [hunk 1]
```diff
@@ -21,3 +21,5 @@
 ### Other redirects
 # Put here redirects for pages that have been moved, etc
 # Add new entries in alphabetical order
+/examples/components/amp-autocomplete: /documentation/examples/interactivity-dynamic-content/simple_autosuggest/
+/examples/components/amp-autocomplete: /documentation/examples/interactivity-dynamic-content/autosuggest_form/
```
Comment (sebastianbenz) [inline platform/config/amp-dev-redirects.yaml]: redirects work the other way around:

```
 /documentation/examples/interactivity-dynamic-content/simple_autosuggest/ -> /examples/components/amp-autocomplete
```

Can you do me a favor and update the documentation to mention this. Thanks!

Comment (caroqliu) [reply platform/config/amp-dev-redirects.yaml]: Should be all set now!

File: platform/config/amp-dev-redirects.yaml [hunk 2]
```diff
@@ -20,4 +21,7 @@
 
 ### Other redirects
 # Put here redirects for pages that have been moved, etc
-# Add new entries in alphabetical order
+# Add new entries in alphabetical order in the format /path/to/old/url: /path/to/new/url
+
+/documentation/examples/interactivity-dynamic-content/autosuggest_form/: /examples/components/amp-autocomplete
```
Comment (sebastianbenz) [inline platform/config/amp-dev-redirects.yaml]: Correct target URL is: `documentation/examples/components/amp-autocomplete`

Comment (caroqliu) [reply platform/config/amp-dev-redirects.yaml]: Ah, sorry. Fixed. :)

**Comment by matthiasrohmer** [comment]
Merged together with #2200.

**Pull request merged**